caravel/templates/manage_listings.html

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.html" %}
+{% block title %} Manage Listings {% endblock %}
+{% block head %}
+  {{ super() }}
+{% endblock %}
+{% block content %}
```

Comment:
Why not add it now? Ajax / thumbnail search (?) seem like relatively independent changes — but maybe I'm misunderstanding something.